spec/controllers/hyrax/batch_ingest/batches_controller_spec.rb

```diff
@@ -20,4 +19,28 @@
       expect(flash[:notice]).to eq "No batch ingest types have been configured."
     end
   end
+
+  describe 'POST /batches/create' do
+    # Mock the nearest edge
+    before { allow(controller).to receive(:start_batch_runner).with(kind_of(Hyrax::BatchIngest::Batch)) }
+
+    context 'with valid params' do
+      let(:batch_params) do
+        { batch: attributes_for(:batch).merge('batch_source' => fixture_file_upload('example_batches/empty.zip')) }
+      end
+
+      it 'calls #start_batch_runner' do
+        post :create, params: batch_params
+        expect(controller).to have_received(:start_batch_runner).with(kind_of(Hyrax::BatchIngest::Batch)).exactly(1).times
+      end
+    end
+
+    context 'with invalid params' do
+      let(:invalid_params) { batch_params.delete(:ingest_type) }
+      xit 'it does not call #start_batch_runner' do
```

Comment:
Do you still want this test skipped?

Reply:
yes, I was going to add additional validations on Batch model, but right now, I think it will still start the batch runner if `ingest_type` is missing or invalid.